packages/react-cosmos-component-playground/src/components/LoadersGrid/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+
+class LoadersGrid extends Component {
```

Comment:
How about naming this something like `ComponentPage`? This page could do many things in the future:
- If there are no fixtures it can be a landing page for that component
- Could display component meta data like name, prop types and description extracted from source comments (e.g. JSDoc)
- Could display a grid of loaders or a list of loaders, a "view modes"